kvmagent/kvmagent/plugins/physical_nic_monitor.py

```
@@ -227,4 +228,9 @@ def physical_nic_monitor(self):
             while True:
                 if time_lock_now != self.time_lock:
                     break
-                self.physical_nic_monitor_get(ip)
+                try:
+                    readable, _, _ = select.select([ip], [], [], 5.0)
+                    if readable:
+                        self.physical_nic_monitor_get(ip)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)